app/views/dashboard_admin/dashboard_admin.html.erb

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dashboard admin</title>
+</head>
+<body>
```

Comment:
Why did you leave this code here? Remember that this part is already in the application.erb

Reply:
Done.